README.rst

```diff
@@ -392,6 +392,76 @@ Example: place the following in your `~/.bashrc`
 .. code:: bash
 
 	SOPS_GPG_EXEC = 'your_gpg_client_wrapper'
+
+Key groups
+~~~~~~~~~~
+
+By default, `sops` encrypts the data key with each of the master keys, such
+that if any of the master keys is available, the file can be decrypted.
+However, it is sometimes desirable to require access to several master keys in
+order to be able to decrypt files. This can be achieved with key groups. With
+key groups, the data key is split into several parts, one for each key group.
+Each key group contains one or more master keys, and all the keys in each
+group encrypt the data key. For decryption, `quorum` parts are required in
+order to retrieve the data key, so at least one key in `quorum` groups has to
+be available.
+
+`quorum` defaults to the number of key groups.
+
+Managing of key groups for SOPS files can be done with the `sops groups`
+command. Key groups can also be specified in the `.sops.yaml` config file,
+like so:
+
+```yaml
+creation_rules:
+    - filename_regex: .*keygroups.*
+      key_groups:
+      # First key group
+      - pgp: fingerprint1,fingerprint2
+        kms: arn1,arn2
+      # Second key group
+      - pgp: fingerprint3,fingerprint4
+        kms: arn3,arn4
+      # Third key group
+      - pgp: fingerprint5,fingerprint6
+        kms: arn5,arn6
+```
```

Comment:
It's not supported, it has to be passed as a command line argument (if not, it will just use the default). I assume this is something we want to support?

Reply:
I think it makes sense to have it in the configuration. It is likely we would want to set the quorum requirement per repo.